docs/README.md

```diff
@@ -177,42 +188,47 @@ To apply CSS customization in this and other blocks, follow the instructions giv
 
 | CSS Handles |
 | ----------- |
-| `installments` |
+| `installmentValue` |
 | `installmentsListContainer` |
 | `installmentsNumber` |
 | `installmentsTotalValue` |
-| `installmentValue` |
+| `installments` |
 | `interestRate` |
 | `listPrice'` |
-| `listPriceRange` |
 | `listPriceRangeMaxValue` |
 | `listPriceRangeMaxWithTax` |
 | `listPriceRangeMinValue` |
 | `listPriceRangeMinWithTax` |
 | `listPriceRangeUniqueValue` |
 | `listPriceRangeUniqueWithTax` |
+| `listPriceRange` |
 | `listPriceValue` |
 | `listPriceWithTax` | 
 | `newPriceValue` |
+| `newSpotPriceValue` | 
 | `paymentSystemName` |
 | `previousPriceValue` |
-| `savings` |
 | `savingsPercentage` |
 | `savingsValue` |
 | `savingsWithTax` |
+| `savings` |
 | `sellingPrice--hasListPrice` |
-| `sellingPrice` |
-| `sellingPriceRange` |
 | `sellingPriceRangeMaxValue` |
 | `sellingPriceRangeMaxWithTax` |
 | `sellingPriceRangeMinValue` |
 | `sellingPriceRangeMinWithTax` |
 | `sellingPriceRangeUniqueValue` |
 | `sellingPriceRangeUniqueWithTax` |
+| `sellingPriceRange` |
 | `sellingPriceValue` |
 | `sellingPriceWithTax` | 
-| `spotPrice` |
+| `sellingPrice` |
+| `spotPriceSavingsPercentage` | 
+| `spotPriceSavingsValue` | 
+| `spotPriceSavingsWithTax` | 
+| `spotPriceSavings` | 
 | `spotPriceValue` |
+| `spotPrice` |
 | `taxPercentage` |
```

Comment:
Can you update the Handles list so it can be organized in alphabetical order? Thanks! 🚀